WalletWasabi.Tests/LiveServerTests.cs

```diff
@@ -0,0 +1,161 @@
+			{
+				var filterModel = IndexDownloader.GetStartingFilter(Network.GetNetwork(networkType.ToString()));
+
+				var filters = await client.GetFiltersAsync(filterModel.BlockHash, 2);
+
+				Assert.True(filters.NotNullAndNotEmpty());
+				Assert.True(filters.Count() == 2);
+			}
+		}
+
+		[Theory]
+		[InlineData(NetworkType.Mainnet)]
+		[InlineData(NetworkType.Testnet)]
+		public async Task GetAllRoundStatesAsync(NetworkType networkType)
+		{
+			using (var client = new SatoshiClient(LiveServerTestsFixture.UriMappings[networkType]))
+			{
+				var states = await client.GetAllRoundStatesAsync();
+				var noRegisteredPeers = states.All(s => s.RegisteredPeerCount == 0);
+
+				Assert.True(noRegisteredPeers);
+				Assert.True(states.NotNullAndNotEmpty());
+				Assert.True(states.Count() == 2);
+			}
+		}
+
+		#endregion Blockchain
+
+		#region ChaumianCoinJoin
+
+		[Theory]
+		[InlineData(NetworkType.Mainnet)]
+		[InlineData(NetworkType.Testnet)]
+		public async Task RegisterAliceInputThenUnConfirmAsync(NetworkType networkType)
+		{
+			(BitcoinPubKeyAddress activeOutputAddress, BitcoinPubKeyAddress changeOutputAddress, string blindedDataHex, string proof, List<TxoRef> utxos) = LiveServerTestsFixture.GetAliceInputData(networkType);
+
+			// blinded activeOutputAddress.ScriptPubKey
+			byte[] blinded = ByteHelpers.FromHex(blindedDataHex);
+
+			var inputProofModels = utxos.Select(txrf => new InputProofModel
+			{
+				Input = txrf,
+				Proof = proof // blinded.BlindedData signed with private key owning utxos
+			});
+
+			var aliceClient = await AliceClient.CreateNewAsync(changeOutputAddress, blinded, inputProofModels, LiveServerTestsFixture.UriMappings[networkType]);
+
+			Assert.NotNull(aliceClient?.RoundId);
+			Assert.NotNull(aliceClient?.UniqueId);
+			Assert.NotNull(aliceClient?.BlindedOutputSignature);
+
+			// need to uncofirm or test will fail when run again
+			await aliceClient.PostUnConfirmationAsync();
+		}
+
+		[Theory]
+		[InlineData(NetworkType.Testnet)]
+		[InlineData(NetworkType.Mainnet)]
+		public async Task RegisterAliceInputThenConfirmAsync(NetworkType networkType)
+		{
+			(BitcoinPubKeyAddress activeOutputAddress, BitcoinPubKeyAddress changeOutputAddress, string blindedDataHex, string proof, List<TxoRef> utxos) = LiveServerTestsFixture.GetAliceInputData(networkType);
+
+			// blinded activeOutputAddress.ScriptPubKey
+			byte[] blinded = ByteHelpers.FromHex(blindedDataHex);
+
+			var inputProofModels = utxos.Select(txrf => new InputProofModel
+			{
+				Input = txrf,
+				Proof = proof // blinded.BlindedData signed with private key owning utxos
+			});
+
+			var aliceClient = await AliceClient.CreateNewAsync(changeOutputAddress, blinded, inputProofModels, LiveServerTestsFixture.UriMappings[networkType]);
+
+			Assert.NotNull(aliceClient?.RoundId);
+			Assert.NotNull(aliceClient?.UniqueId);
+			Assert.NotNull(aliceClient?.BlindedOutputSignature);
```

Comment:
Same here, remove the null coalescing operators.

Reply:
Done.